src/tools/jobs/stats-territoires.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -l
+
```

Comment:
@MorganP-projects Il faut que ce fichier soit avec les permissions 100755 pour que le cron puisse fonctionner, est-bien le cas ?

Reply:
Je ne trouve pas la commande pour ces droits ci, c'est équivalent à quel chmod ?

Reply:
il me semble que tu fais juste un chmod +x dessus et le git diff/git commit va le détecter

Reply:
Je faisais un git update-index --chmod=+x path/to/file